### PR TITLE
Libxml++

### PR DESCRIPTION
**Pull request by scopatz**
Hope this works...

### CODE REVIEW

**Comment by scopatz** [comment]
Wow! This built on the first try...

File: recipes/libxmlpp/build.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  export CXXFLAGS="${CXXFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export CXXFLAGS="${CXXFLAGS} -stdlib=libc++"
+  export LDFLAGS="${LDFLAGS} -mmacosx-version-min=${MACOSX_VERSION_MIN}"
+  export LDFLAGS="${LDFLAGS} -stdlib=libc++ -lc++"
+  export LINKFLAGS="${LDFLAGS}"
+fi
+
+./configure --prefix="${PREFIX}" || { cat config.log; exit 1; }
+make
+make install
```
Comment (ocefpaf) [inline recipes/libxmlpp/build.sh]: Is there a `make check`?

Comment (scopatz) [inline recipes/libxmlpp/build.sh]: Yep, trying it now.

**Comment by scopatz** [comment]
Ok, I can run `make check` on Linux, but Mac fails for some reason probably related to paths and linking.  I have disabled check on Mac but left it in on Linux. It is worth noting that homebrew doesn't run make check, either. Hopefully, this is good enough.

**Comment by ocefpaf** [comment]
> Hopefully, this is good enough.

:+1: 

Let's just open an issue in the feedstock so we can remember to address this later. (Maybe [this](https://github.com/conda-forge/nco-feedstock/blob/master/recipe/build.sh) pattern can solve it.)

**Comment by scopatz** [comment]
Thanks @ocefpaf! Sounds like a plan.

**Pull request merged**